external-crates/move/crates/move-cli/tests/sandbox_tests/print_stack_trace/args.exp

```diff
@@ -1,7 +1,23 @@
 Command `sandbox publish`:
 Command `sandbox run storage/0x0000000000000000000000000000000000000000000000000000000000000042/modules/print_stack_trace.mv print_stack_trace --dry-run`:
 Call Stack:
-    [0] 0000000000000000000000000000000000000000000000000000000000000042::print_stack_trace::print_stack_trace
+    [0] 0000000000000000000000000000000000000000000000000000000000000002::M::sum
```

Comment:
This test was weird because the actual print happens in `sum`, but the previous arrangement of trace printing ignored the current frame of the VM.